docs/platforms/apple/guides/ios/session-replay/index.mdx

```diff
@@ -73,7 +73,9 @@ Sampling begins as soon as a session starts. `sessionSampleRate` is evaluated fi
 
 ## Privacy
 
-The SDK is recording and aggressively redacting all text and images. We plan to add fine controls for redacting, but in this version, we just allow either on or off. The default is on. Please don’t turn it off if you have sensitive data in your app. Before the Beta is complete, we'll give you the controls you need.
+The SDK is recording and aggressively redacting all text and images. 
+Please don't turn it off if you have sensitive data in your app. 
+If you want to manually choose which part of your app's data to redact, read our guide on [custom redaction](/platforms/apple/guides/ios/session-replay/customredact).
```

Comment:
```suggestion
The SDK aggressively records and redacts all text and images. 
Please don't turn it off if you have sensitive data in your app. 
If you want to manually redact parts of your app's data, read our guide on [custom redaction](/platforms/apple/guides/ios/session-replay/customredact).
```

Reply:
Not sure about this one.

This is what I meant:
`The SDK is recording your app, and aggressively redacting all text and images.`

We dont record text and images